src/nvim/shada.c

```diff
@@ -2318,6 +2318,55 @@ static inline ShaDaWriteResult shada_read_when_writing(
   return ret;
 }
 
+/// Get list of buffers to write to the shada file
+///
+/// @param[in] khash_t(bufset)   Buffers which are ignored
+///
+/// @return ShadaEntry  kSDItemBufferList of buffers to save
```

Comment:
Do not remember seeing type after return. I would write something like “list of buffers to save[, kSDItemBufferList entry].”: type is known from function signature.